site/src/routes/blog/[slug].json.js

```diff
@@ -1,22 +1,23 @@
-import send from '@polka/send';
 import get_posts from './_posts.js';
 
 let lookup;
 
-export function get(req, res) {
+export function get({ params }) {
 	if (!lookup || process.env.NODE_ENV !== 'production') {
```

Comment:
Is it safe to be referring to `process.env.NODE_ENV` here (and a number of other places) in code that's going to be processed by Vite? Should we be using `process.env['NODE_ENV']` or is that only necessary in `.svelte` files?

Reply:
Oh, right, yes there is special handling for `process.env.NODE_ENV` that kicks in before other `process.env.` handling happens.

https://github.com/vitejs/vite/blob/main/packages/vite/src/node/plugins/define.ts